skunk/rust/skunk/src/ir_gen/state_values.rs

```diff
@@ -305,7 +294,7 @@ impl <'ctx> StateValue<'ctx> {
       ValueParts::SimplePrimitive(value) => value.get_type(),
       ValueParts::CompoundPrimitive(value) => value.get_type().into(),
       ValueParts::DynamicMemRegion(ptr, _) | ValueParts::StaticMemRegion(ptr) => ptr.get_type().into(),
-      ValueParts::None | ValueParts::Suppress => panic!("dunno!"),
+      ValueParts::None => panic!("dunno!"),
     }
```

Comment:
At some point we should audit the TODOs and panics and convert to issues where needed

Reply:
Yup. BTW / aside: I've actually been converting some of the Result errors back to panics (where it's a should-not-happen style thing) because the bugs that triggering these uncover are easier to find that way.